pkg/devcontainers/features/oci_test.go

```
@@ -0,0 +1,744 @@
+		}
+	})
+}
+
+// --- fetchBearerToken ---
+
+func TestFetchBearerToken_SuccessTokenField(t *testing.T) {
+	t.Parallel()
+
+	srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-Type", "application/json")
+		fmt.Fprint(w, `{"token":"mytoken123"}`)
```

golangci-lint (2.11.4)

[error] Error return value of `fmt.Fprint` is not checked

(errcheck)